lib/specberus-wrapper.js

```diff
@@ -1,118 +1,123 @@
+/* eslint-disable global-require */
```

Comment:
This is now deprecated in ESLint 7: https://eslint.org/docs/rules/global-require
Since we already have `eslint-plugin-node`, please use https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/global-require.md